robotcontainer.py

```diff
@@ -41,19 +42,20 @@
 from commands.shootball import ShootBall
 from commands.defensestate import DefenseState
 
-from commands.indexer.holdball import HoldBall
 from commands.intake.autoballintake import AutoBallIntake
 from commands.intake.deployintake import DeployIntake
 from commands.intake.retractintake import RetractIntake
 from commands.shooter.aimshootertotarget import AimShooterToTarget
 from commands.shooter.aimshootermanual import AimShooterManually
-from commands.shooter.tarmacshot import TarmacShot
 from commands.shooter.stopaimsystem import StopMovingParts
 
 from commands.auto.fivebrstandard import FiveBRStandard
 from commands.auto.fourblnoninvasive import FourBLNoninvasive
 from commands.auto.twoblhangerouttake import TwoBLHangerOuttake
 from commands.auto.threebrstandard import ThreeBRStandard
+from commands.shooter.decreaseshooterspeed import DecreaseShooterSpeed
+from commands.shooter.increaseshooterspeed import IncreaseShooterSpeed
+from commands.shooter.resetshooteroffset import ResetShooterOffset
```

Comment:
Should these be moved up to line 50?

Reply:
Yes, let me do that.  Weird that this movement was good enough to silence the error